exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -214,17 +214,38 @@ class Tracer : public opentelemetry::trace::Tracer,
     {
       size_t idx = 0;
       std::string linksValue;
+
+      // reserve space for all the SpanLinks.
+      // A single SpanLink will be outptut as:
+      // [{"toSpanId":"9a43c801557f26b7","toTraceId":"ac6cd70ac4bb168a99cb7651b048d965"}]
+      // The second and above link output to string are 1 byte less than the first link.
+      const size_t linkValueSize = 80;
```

Comment:
the max allowed link size is 128, as per the specs - https://github.com/open-telemetry/opentelemetry-specification/blob/625e0f8500f2aea48982a943997c991ff41ee9bb/specification/trace/sdk.md#span-limits